alembic.ini

```diff
@@ -93,4 +93,4 @@ formatter = generic
 
 [formatter_generic]
 format = %(levelname)-5.5s [%(name)s] %(message)s
-datefmt = %H:%M:%S
+datefmt = %H:%M:%S
```

Comment:
This is probably an unintended change in this commit.

Reply:
Yes, please remove this change.